loginFunctions.js

```diff
@@ -0,0 +1,103 @@
+const ERRORMESSAGE = {
+  wrongEmail: "잘못된 이메일 형식입니다",
+  wrongPassword: "비밀번호를 8자 이상 입력해주세요",
+  emailIsEmpty: "이메일을 입력해주세요",
+  passwordIsEmpty: "비밀번호를 입력해주세요",
+  nicknameIsEmpty: "닉네임을 입력해주세요",
+  passwordIsNotSame: "비밀번호가 일치하지 않습니다.",
+};
```

Comment:
💊 제안
반복사용되는 에러메시지를 따로 상수로 모아두신 것 좋습니다. 다만 단어를 구분할 수 있게 아래처럼 명명하시는 것을 추천드려요!

```suggestion
const ERROR_MESSAGE = {
  wrongEmail: "잘못된 이메일 형식입니다",
  wrongPassword: "비밀번호를 8자 이상 입력해주세요",
  emailIsEmpty: "이메일을 입력해주세요",
  passwordIsEmpty: "비밀번호를 입력해주세요",
  nicknameIsEmpty: "닉네임을 입력해주세요",
  passwordIsNotSame: "비밀번호가 일치하지 않습니다.",
};
```